lib/katello.rb

```diff
@@ -12,6 +12,7 @@
 require "rest_client"
 require "anemone"
 require "securerandom"
+require "strong_parameters"
```

Comment:
Has this been tested? I realize that all the tests won't pass when running the test suite but if the requires statement blows up we need to yank it out.